README.md

```diff
@@ -137,7 +132,7 @@ usage: sigstore sign [-h] [-v] [--identity-token TOKEN] [--oidc-client-id ID]
                      [--oauth-force-oob] [--no-default-files]
                      [--signature FILE] [--certificate FILE] [--bundle FILE]
                      [--output-directory DIR] [--overwrite] [--staging]
-                     [--rekor-url URL] [--rekor-root-pubkey FILE]
+                     [--rekor-url URL]
                      [--fulcio-url URL] [--ctfe FILE]
```

Comment:
I believe we also want to get rid of `--ctfe`, since it's also loaded from the TrustedRoot.